src/Microsoft.DotNet.Darc/src/DarcLib/IRemote.cs

```diff
@@ -265,6 +267,14 @@ public interface IRemote
 
         #region Repo/Dependency Operations
 
+        /// <summary>
+        /// Get the tools.dotnet section of the global.json from a target repo URI
```

Comment:
Can you fix up the comments here (I was just copying/pasting when unblocking the source build folks)